trendly/src/app/clusterly/clusterly.component.ts

```diff
@@ -309,38 +314,37 @@ export class ClusterlyComponent implements OnInit {
   }
 
   /** Applies given simulation on inner and outer circles */
-  private applySimulation(simulation, circle, lightCircle): void {
-    simulation
-      .nodes(this.queries)
-      .on('tick', () => {
-        circle
-          .attr('cx', (d) => d.x)
+  private applySimulation(): void {
+    this.simulation.nodes(this.queries).on('tick', () => {
+      this.circles.attr('cx', (d) => d.x)
           .attr('cy', (d) => d.y)
-        lightCircle
-          .attr('cx', (d) => d.x)
+          .style(
+              'fill', (d) => this.scales.get(Scales.ColorScale)(d.clusterId));
+      this.lightCircles.attr('cx', (d) => d.x)
           .attr('cy', (d) => d.y)
-      });
+          .style(
+              'fill',
+              (d) => this.scales.get(Scales.LightColorScale)(d.clusterId))
+    });
   }
 
   /** Adds dragging functionality to inner and outer circles */
-  private applyDragging(
-      simulation, tooltip, circle, lightCircle,
-      clusterIdToLoc: Map<number, Location>): void {
+  private applyDragging(tooltip, clusterIdToLoc: Map<number, Location>): void {
     // A pointer to ClusterlyComponent instance
     const clusterly = this;
 
-    circle.call(d3.drag()
-                    .on('start', dragstarted)
-                    .on('drag', dragged)
-                    .on('end', dragended));
-    lightCircle.call(d3.drag()
-                         .on('start', dragstarted)
-                         .on('drag', dragged)
-                         .on('end', dragended));
+    this.circles.call(d3.drag()
+                          .on('start', dragstarted)
+                          .on('drag', dragged)
+                          .on('end', dragended));
+    this.lightCircles.call(d3.drag()
+                               .on('start', dragstarted)
+                               .on('drag', dragged)
+                               .on('end', dragended));
 
     function dragstarted(event, d): void {
```

Comment:
use camelCase for method / variable names, also below.
```suggestion
    function dragStarted(event, d): void {
```